src/parser.cpp

```diff
@@ -393,12 +403,12 @@ void Parser::ParserImpl::loadComponent(const ComponentPtr &component, const XmlN
             ResetPtr reset = std::make_shared<Reset>();
             loadReset(reset, component, childNode);
             component->addReset(reset);
-        } else if (childNode->isType("math")) {
+        } else if (childNode->isType(MATHML_NS, "math")) {
             // TODO: copy any namespaces declared in parents into the math element
```

Comment:
Are we still going down this "convert-to-text" route? I note the validator just uses libxml. If we use libxml to parse the mathml this TODO can go...

Reply:
Could you remind me what this "convert-to-text" route is all about?